### PR TITLE
REL-931516 Server 2023 Patch 2 - Update RAPCI repos to upgrade hopper with hotfix/patch.

### PR DESCRIPTION
**Pull request by shivarajkamadod**
- Propagating patch1 changes to server release 2023
- [REL-931516](https://jira.kcura.com/browse/REL-931516) Server 2023 Patch 2 - Update RAPCI repos to upgrade hopper with hotfix/patch.
- 
image

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,12 +2,38 @@
 All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
+## [23000.19.1004] - 01-APR-2024
```
Comment (ravikonnur) [inline CHANGELOG.md:5]: Update the date to May month

File: README.md [hunk 2]
```diff
@@ -27,10 +27,11 @@ For more information on RAP CD, [view the documentation in Einstein](https://ein
 
 ## History
 
-The server-release-12.3-2023 branch was migrated from https://git.kcura.com/projects/DTX/repos/relativity-data-transfer-legacy-sdk/browse?at=refs%2Fheads%2Fserver-release-12.3-2023 by the Server Data Transfer Team. 
-Tag: 23000.19.1-server 
-Branch: server-release-12.3-2023
+The server-release-12.3-2023-patch1 branch was migrated from https://git.kcura.com/projects/DTX/repos/relativity-data-transfer-legacy-sdk/browse?at=refs%2Fheads%2Fserver-release-12.3-2023-patch1 by the Server Data Transfer Team.
```
Comment (ravikonnur) [inline README.md:30]: This information should not be changed

Comment (shivarajkamadod) [reply README.md]: Done

File: CHANGELOG.md (hunk 1 above)
Comment (shivarajkamadod) [reply CHANGELOG.md]: Done

**Review by ravikonnur — APPROVED** [review]

**Review by ravira73 — APPROVED** [review]

**Review by Hemashree20 — APPROVED** [review]

**Pull request merged**